harpy/commands/qc.py

```
@@ -2,12 +2,13 @@
 
 import os
 import rich_click as click
-from harpy.common.cli_types_generic import HPCProfile, MultiInt, SnakemakeParams
+from harpy.validation.fastq import FASTQ
+from harpy.validation.fasta import FASTA
```

Ruff (0.12.2)

`harpy.validation.fasta.FASTA` imported but unused

Remove unused import: `harpy.validation.fasta.FASTA`

(F401)